### ISSUE TITLE
Don't run amon as root during installation

### ISSUE THREAD

**Issue by bascht** [amonapp/amon-docs#1]
_(I hope it's okay if I grab issue #1 ;-) Congrats to open sourcing Amon! Very much looking forward to it!)_

The [install docs](https://github.com/amonapp/amon-docs/blob/master/docs/onpremise.md#setting-up-an-environment) aren't totally clear which user the installation should be run as. I suggest switching to the amon user right after chowning the folders. 

Otherwise this command

```
# To test if Amon is configured properly
python manage.py runserver
```

will create files as root in `/opt/amon` and the regular systemd installation won't come up. :)

**Comment by martinrusev** [comment]
@bascht  Yep. Running with `su - amon -c "python manage.py  runserver"` or something similar will be a much better solution
